Mail/Views/Thread/Message/MessageBodyView.swift

```diff
@@ -72,6 +78,27 @@ struct MessageBodyView: View {
                 ShimmerView()
             }
         }
+        .onReceive(printNotif) { _ in
```

Comment:
Is it possible to write something like `.onReceive(printNotificationPublisher, perform: printMessage)`